cmd/state-update-dialog/app.go

```diff
@@ -30,13 +32,13 @@ type App struct {
 func NewApp() *App {
 	a := &App{}
 	a.wails = wails.CreateApp(&wails.AppConfig{
-		Width:            600,
-		Height:           400,
-		Title:            "ActiveState Desktop - Update Available",
-		HTML:             html,
-		JS:               js,
-		CSS:              css,
-		Colour:           "#FFF", // Wails uses this to detect dark mode
+		Width:  800,
+		Height: 600,
```

Comment:
```suggestion
		Width:  600,
		Height: 400,
```

Don't worry about the design for this PR.